src/render/texture.js

```diff
@@ -40,35 +40,51 @@ class Texture {
     format: TextureFormat;
     filter: ?TextureFilter;
     wrap: ?TextureWrap;
+    useMipmap: boolean;
+    powerOfTwoSize: boolean;
 
-    constructor(context: Context, image: TextureImage, format: TextureFormat, premultiply: ?boolean) {
+    constructor(context: Context, image: TextureImage, format: TextureFormat, premultiply: ?boolean, useMipmap: ?boolean) {
```

Comment:
Rather than add a second optional boolean parameter, let's move `premultiply` and `useMipmap` into an options object so they get descriptive names at the call site and there's no need for an `undefined` placeholder argument.